packages/config/config/devices/0x018e/sdco.json

```
@@ -13,6 +13,11 @@
 			"productType": "0x0003",
 			"productId": "0x0015",
 			"zwaveAllianceId": 3670
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x001a",
+			"zwaveAllianceId": 3724
```

```suggestion
			"productId": "0x001a"
```
This ZWA ID belongs to the one with product ID 0x0014